src/asciidoc/docs/develop/workLocally.adoc

```diff
@@ -217,33 +217,10 @@ npm run start-windows
 ==== Launching the backend
 
 The backend needs to have the values of the `KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables to access the leaderboard database.
-You get the environment variables that Koji uses to deploy your project from the Koji editor.
+You saved the environment variables that Koji uses to deploy your project before you cloned it.
+If you need to find the environment variables again, follow the instructions in <<use-git#_obtaining_koji_environment_variables,Obtaining Koji environment variables>>.
 
-. Open your project in the Koji editor.
-. Open a new terminal tab and run the following command.
-+
-[source,bash]
-env | grep 'KOJI_'
-+
-Your terminal will look something like this:
-+
-[source,bash]
-----
-root@ip-172-31-15-24:/usr/src/app# env | grep 'KOJI_'
-...
-KOJI_SERVICE_URL_backend=https://3333-48006672-6558-4f69-a40c-e4142c15067f.koji-staging.com
-
-... (some variables not shown) ...
-
-KOJI_PROJECT_ID=a70f8329-e89e-48b0-8d85-7658c1542b9f
-KOJI_PROJECT_TOKEN=6679483a-dab8-4e89-9a83-6b56b53b4241
-...
-----
-NOTE: This example was edited to show only the relevant items.
-The values in your output will be unique to your project.
-
-. Copy the output for the `KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN`.
-. On your local machine, create a file named `.env` at the root of your project, and paste the two lines that you have just copied into it.
+. On your local machine, create a file named `.env` at the root of your project, and paste the two lines for the environment variables.
 +
 Your file will look something like this (with your unique values):
 +
```

Comment:
Looks like there was an errant period in the command to launch the backend. Can you remove it?